back/sssscs/src/main/java/com/ib/config/StandardConfig.java

```diff
@@ -0,0 +1,21 @@
+package com.ib.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.config.annotation.authentication.configuration.AuthenticationConfiguration;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@Configuration
+public class StandardConfig {
+	@Bean
+	public PasswordEncoder encoder() {
+		return new BCryptPasswordEncoder();
+	}
+	
+    @Bean
+    public AuthenticationManager authenticationManager(AuthenticationConfiguration authConfig) throws Exception {
+        return authConfig.getAuthenticationManager();
+    }
```

Comment:
Spaces used for indentation instead of tabs

Reply:
Fixed in [438f479](https://github.com/magley/sssscs/pull/1/commits/438f4790627e305b6f659d6cc8ef0a009b94ac69).